scripts/tools/release-cli/release-cli.js

```diff
@@ -68,6 +67,11 @@ const installPaths = {
   testing_utilities: path.resolve(base, 'client-frameworks-support', 'testing-utilities')
 };
 
+if (process.env.NIGHTLY === 'true') {
```

Comment:
another (hopefully :D ) last edge case that I could think of is, if we run an [RC Release](https://github.com/SAP/luigi/blob/main/.github/workflows/rc-release.yml) for Core/Client/etc :

RC Releases are a modified version of a nightly, but released not nightly but [on request](https://github.com/SAP/luigi/actions/workflows/rc-release.yml), that we do only for Core/Client/etc, **not** Container.  

For RC Releases **process.env.NIGHTLY** needs to be '**true**' but also **process.env.NIGHTLY_VERSION** needs to be set with a (string) version , so only checking `NIGHTLY === 'true' ` might result in releasing an RC Release for Container without intending to do so. Container RC Releases are done manually on a separate process. 

So to fix that issue I think we can do:
`if (process.env.NIGHTLY === 'true' &&  process.env.NIGHTLY_VERSION is NOT set  )`

And I think we can add the same **if check** on **scripts/tools/publish-nightly.js**, just to be on the safe side, so we don't let it do RC release on Container. (For : publishPaths, packagePaths)